libyul/backends/evm/StackHelpers.h

```diff
@@ -254,6 +254,18 @@ class Shuffler
 					ops.isCompatible(sourceTop, offset) // The lower position wants to have this slot.
 				)
 				{
+					// We cannot swap that deep.
+					if (ops.sourceSize() - offset - 1 > 16)
```

Comment:
Can this underflow?

Reply:
Nope, offset is at most ``ops.sourceSize() - 1`` since it's in ``ranges::views::iota(0u, std::min(ops.sourceSize(), ops.targetSize())``